[Bug report] CAU = 0.3.5 does‘t work with SDK <= 3.7.6

```
Packages\com.anatawa12.continuous-avatar-uploader\Editor\Uploader.cs(570,26): error CS0117: 'VRCApi' does not contain a definition for 'ContentUploadConsent'

Packages\com.anatawa12.continuous-avatar-uploader\Editor\Uploader.cs(570,51): error CS0246: The type or namespace name 'VRCAgreement' could not be found (are you missing a using directive or an assembly reference?)
```

--- a/CHANGELOG-PRERELEASE.md
+++ b/CHANGELOG-PRERELEASE.md
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog].
 ### Removed
 
 ### Fixed
+- Incompatibility with older VRCSDK `#96`
 
 ### Security
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog].
 ### Removed
 
 ### Fixed
+- Incompatibility with older VRCSDK `#96`
 
 ### Security
 

--- a/Editor/Uploader.cs
+++ b/Editor/Uploader.cs
@@ -558,6 +558,7 @@ namespace Anatawa12.ContinuousAvatarUploader.Editor
         public static string AgreementText =
             "By clicking OK, I certify that I have the necessary rights to upload this content and that it will not infringe on any third-party legal or intellectual property rights.";
 
+#if CAU_VRCSDK_BASE_3_8_0
         private static async Task AddCopyrightAgreement(string blueprint)
         {
             const string key = "VRCSdkControlPanel.CopyrightAgreement.ContentList";
@@ -575,6 +576,9 @@ namespace Anatawa12.ContinuousAvatarUploader.Editor
                 Version = 1,
             });
         }
+#else
+        private static Task AddCopyrightAgreement(string blueprint) => Task.CompletedTask;
+#endif
 
         /// <summary>
         /// This class sets the blueprintId every frame until disposed.

--- a/Editor/com.anatawa12.continuous-avatar-uploader.editor.asmdef
+++ b/Editor/com.anatawa12.continuous-avatar-uploader.editor.asmdef
@@ -26,6 +26,11 @@
             "name": "com.vrchat.base",
             "expression": "3.6.0",
             "define": "CAU_VRCSDK_BASE_3_6_0"
+        },
+        {
+            "name": "com.vrchat.base",
+            "expression": "3.8.0",
+            "define": "CAU_VRCSDK_BASE_3_8_0"
         }
     ],
     "noEngineReferences": false

